src/cli/Clip.cpp

```diff
@@ -15,18 +15,14 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include <chrono>
-#include <cstdlib>
-#include <thread>
-
 #include "Clip.h"
 
-#include "cli/TextStream.h"
-#include "cli/Utils.h"
-#include "core/Database.h"
+#include "Utils.h"
 #include "core/Entry.h"
-#include "core/Global.h"
 #include "core/Group.h"
+#include "core/Tools.h"
+
+#define DEFAULT_TIMEOUT 10
```

Comment:
@droidmonkey since this is going to be global, maybe we can give it a more explicit name, like `DEFAULT_CLIP_TIMEOUT`